### PR TITLE
docs: tip on shut workflow on in forked repos

### PR DESCRIPTION
**Pull request by xingxing**
## Documentation (please check one with an [x])

- [X] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [X] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: .github/contributing.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ If you've found a **bug** or have a **feature request** then please create an is
 ## Code
 
 If you would like to fix a bug or work on a feature, please fork the repository and create a Pull Request.
+And also please remember to shut down all GitHub Action workflows in your forked repository except you accucally need to use them.
```
Comment (rarkins) [inline .github/contributing.md:22]:
```suggestion
And also please remember to shut down all GitHub Action workflows in your forked repository except you actually need to use them.
```

**Comment by rarkins** [comment]
I'm confused why this only happened to @xingxing because we've never seen it with other users before. At first I suspected it might be because you were committing to `main` branch but your open PR comes from non-main. Maybe it happened when you sync'd your `main` branch from upstream, but then again I assume you're not the only person to have done this.

Here's the run:

https://github.com/xingxing/renovate/actions/runs/3088182002

**Comment by xingxing** [comment]
> I'm confused why this only happened to @xingxing because we've never seen it with other users before. At first I suspected it might be because you were committing to `main` branch but your open PR comes from non-main. Maybe it happened when you sync'd your `main` branch from upstream, but then again I assume you're not the only person to have done this.
> 
> Here's the run:
> 
> https://github.com/xingxing/renovate/actions/runs/3088182002

I sync'd the `main` branch of my forked repo by licking the sync button on the page.
Screen Shot 2022-09-20 at 8 13 53 PM

**Comment by HonkingGoose** [comment]
Copy/pasting my comment from #17873:

It sounds like a failing workflow on @xingxing's _forked_ repository results in a warning on our repository. This is really weird. I would expect `semantic-release` to open the warning issue on the _fork_.

It almost sounds like a bug in `semantic-release`. But before I go bother the `semantic-release` maintainers, let's see if we can figure out what's happening? 😉

**Comment by travi** [comment]
the reason that semantic-release is posting the issue back to this repository is most likely because the `repository.url` is still pointed at this repository in the fork: https://github.com/xingxing/renovate/blob/a1967593706d60c7ffe5a85e953789bc238f110c/package.json

i'm not sure why this would have been the first case where you've seen this happen, though.

**Comment by rarkins** [comment]
Perhaps this would prevent it from running? https://github.com/renovatebot/renovate/pull/18101/files

**Comment by rarkins** [comment]
Can we possibly add logic to our workflows so that they skip execution if not running in this repo?

**Pull request closed without merging**